frontend-react/src/pages/data-dashboard/DataDashboard.tsx

```diff
@@ -0,0 +1,56 @@
+import { GridContainer } from "@trussworks/react-uswds";
+import { Helmet } from "react-helmet-async";
+import React from "react";
+
+import { FeatureName } from "../../AppRouter";
+import { AuthElement } from "../../components/AuthElement";
+import { MemberType } from "../../hooks/UseOktaMemberships";
+import { USNavLink } from "../../components/USLink";
+import { withCatchAndSuspense } from "../../components/RSErrorBoundary";
+
+import DataDashboardTable from "./DataDashboardTable/DataDashboardTable";
+
+function DataDashboard() {
+    return (
+        <div className="rs-data-dashboard">
+            <header className="usa-header usa-header--extended bg-primary-darker text-white">
+                <GridContainer className="margin-left-7 margin-right-7 padding-top-8 padding-bottom-8 rs-max-width-100-important">
+                    <div className="font-sans-lg text-blue-30 width-full">
+                        Colorado Department of Public Health and Environment
```

Comment:
Sorry if I don't have context here, but should `Colorado Department of Public Health and Environment` be hard coded here? Or should whatever org they have selected be appearing here?

Reply:
Good catch!